platform/viewer/src/routes/WorkList/WorkList.jsx

```diff
@@ -45,6 +45,7 @@ function WorkList({ history, data: studies, isLoadingData, dataSource }) {
   const debouncedFilterValues = useDebounce(filterValues, 200);
   const { resultsPerPage, pageNumber, sortBy, sortDirection } = filterValues;
 
+  let dynamicFilterValues = {};
```

Comment:
I'm not sure I understand why this variable has this name. Isn't this just the _default_ sort values?

Reply:
@swederik We already have the default filter values, I called it dynamic because it's based on a runtime conditional inside the sort. I'm open to suggestions, naming is hard. 😄

Reply:
maybe runtimeFilterValues?

Reply:
Call it defaultSortValues and just make a comment that they are only applied if no other sorting is specified, then you can merge it